module/library/individualAssessment/helper.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Individual library helper functionality.
```

Comment:
@aman-tunerlabs - Description should be - "Individual assessment  solutions library helper functionality."